drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -212,9 +212,14 @@ function DragAndDropTemplates(configuration) {
             gettext('Items placed here: ') + items_in_zone.map(function (item) { return item.displayName; }).join(", ")
           );
         }
+
+        /* Force zone alignment if showing answer. */
+        if (ctx.show_answer && zone.align === 'none') {
+          zone.align = 'center'
+        }
+
         if (zone.align !== 'none') {
           item_wrapper += '.item-align.item-align-' + zone.align;
-          //items_in_zone = $.grep(ctx.items, is_item_in_zone);
```

Comment:
@arbrandes I wasn't talking about leaving the comment in, I was wondering if the code itself was still needed. The `else` branch below modifies `items_in_zone`, but the `if` branch currently doesn't. That asymmetry looked suspicious to me.

So, another way of putting the question would be: Why does the `if` branch get away with not modifying `items_in_zone`?